libs/libcommon/src/libcommon/config.py

```diff
@@ -360,6 +360,8 @@ class ProcessingGraphConfig:
                 "input_type": "split",
                 "triggered_by": [
                     "config-split-names-from-info",
+                    "config-split-names-from-streaming",
+                    "config-parquet-and-info",
```

Comment:
But if config-split-names-from-info finishes with ResponseAlreadyComputedError it will not compute the response, right? Even if config-split-names-from-streaming was a success.

Reply:
Why would it not compute the response?

Error and success trigger the children steps.

And inside the split-duckdb-index job runner, you get the best response from both steps, so: it will compute the response with the valid response

https://github.com/huggingface/datasets-server/blob/f9d5c9ff873ab93369ec605644b5e72f595ac6e2/services/worker/src/worker/job_runners/split/duckdb_index.py#L70-L72